src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/SchoolYearValidation.feature

```diff
@@ -0,0 +1,208 @@
+@SchoolYear
+Feature: School Year Reference Validation
+  Reference validation on School Years
+
+  """
+  @StudentEducationOrganizationAssociation Hook that will create the following data for the required tests
+  createStudent(studentUniqueId)
+  createSchool(schoolId)
+  createCourse(courseCode, schoolId)
+  createSession(schoolId, schoolYear, sessionName)
+  createCourseOffering(localCourseCode, courseCode, schoolId, schoolYear, sessionName)
+  createSection(localCourseCode, schoolId, schoolYear, sessionName, sectionIdentifier)
+  """
+
+  @Ignore @StudentEducationOrganizationAssociation
+  Scenario: Post valid request using a existing year and make sure response is correctly
```

Comment:
Suggestion: "Try creating a resource using a valid school year".

No need to mention "response is correct" - we can take that as a given.

Reply:
Done